element_array_ephys/ephys_acute.py

```diff
@@ -180,7 +180,10 @@ def auto_generate_entries(cls, session_key):
                     "probe_type": spikeglx_meta.probe_model,
                     "probe": spikeglx_meta.probe_SN,
                 }
-                if probe_key["probe"] not in [p["probe"] for p in probe_list]:
+                if (
+                    probe_key["probe"] not in [p["probe"] for p in probe_list]
+                    and probe_key not in probe.Probe()
```

Comment:
No need for this second check because we opted for `skip_duplicates=True` when inserting below

Reply:
Oh this is the result of a merge-conflict resolution. I'll remove it